### ISSUE TITLE
iOS: Unable to Connect to Dongle - get's hung on "Waiting for bluetooth deice..." 

### ISSUE THREAD

**Issue by bobfortin** [koush/vysor.io#999]
**To Reproduce**
1. Installed Dongle (MakerDiary w. Case)
2. Launch Vysor
3. Connect iPhone 8
4. Vysor recognized device
5. Select Play
6. Vysor window launched for iPhone 8
7. Hangs on "Waiting for bluetooth device..."
8. Bug Report: https://gist.github.com/VysorLogs/def7498ddecc2b91b760b214112a142d

**Expected behavior**
Vysor completes connection to device and I can interact with the iOS device

**Screenshots**
Screenshot: https://www.screencast.com/t/8pqjca49bNr0
Video: https://www.screencast.com/t/tex5vnbQv

**Desktop (please complete the following information):**
 - OS: MacOS 12.1 (Intel)

**iOS (please complete the following information):**
 - Device: iPhone 8
 - OS: iOS 14.4
 - Vysor Bug Report Link: https://gist.github.com/VysorLogs/def7498ddecc2b91b760b214112a142d

**iOS Bluetooth Issues (optional):**
 - MBP 16 Inch (2019)
 - Apple Keyboard / Mouse
 - No Bluetooth devices connected to iOS device

**Additional context**
1. I have tried to Reset Vysor
2. I have tried to set Settings to Default
3. I have tried to uninstall / reinstall Vysor

Having same issue when attempting to use the following devices:
iPhone SE / iOS 13.x
iPhone 12 / iOS 15.x

**Comment by koush** [comment]
Thanks, did you try unplug and replugging the dongle per my last instructions in the other issue?

**Comment by bobfortin** [comment]
Yes sir, I have tried unplugging and plugging the Dongle back in.

**Comment by koush** [comment]
I am realizing the instructions are incorrect for dongle setup. When you are prompted to pair with your computer, you actually want to pair with your NRF Dongle. Reset to default again, and initiate pairing from your phone, to the dongle.

![image](https://user-images.githubusercontent.com/73924/146841560-dae9e36d-326c-4d3d-92db-4c971508e74d.png)

**Comment by koush** [comment]
Maybe you did that the first time around, but it's just something I noticed while going through the process myself. And pairing from your computer would explain why it's stuck on connecting.

**Comment by bobfortin** [comment]
Not seeing the Dongle on Bluetooth...  any device previously mentioned.
Yes, have a Green LED Light on the Dongle
https://www.screencast.com/t/Bxc7JXtu1J

**Comment by koush** [comment]
Ok, you did see the prompt in vysor to pair with iPhone right? At that point is when the dongle should become visible to your iOS device.

**Comment by bobfortin** [comment]
90s Video: https://www.screencast.com/t/kzABmzXGp3

**Comment by koush** [comment]
<img width="341" alt="image" src="https://user-images.githubusercontent.com/73924/146844019-cb96a180-f2b4-422b-90b4-ead2aedb1688.png">

Unplug/replug the dongle. Refresh. If it asks to flash an update, do that. I'm not sure if you have done that yet.

**Comment by bobfortin** [comment]
Yes, I've done each of those steps several times - always comes back to Flash Update.

I've tried another Mac with exactly the same results.

Flash File name: zephyr.uf2

**Comment by koush** [comment]
This is strange, because in a prior screenshot, it was not showing the "Flash Update" option, indicating it was functioning properly.

https://www.screencast.com/t/PPEpg4t0p6vv

**Comment by bobfortin** [comment]
Sorry, not sure what to tell you - I've tried everything you've asked of me today and it has not worked.  

Spent $25 to get this to work because of Bluetooth issues with Monterey - and can't get it to work on multiple devices and computers.  

Back to the old days - mirroring using Quicktime.

Thanks for the effort and responses.

**Comment by jillomonster** [comment]
I have been getting this issue consistently for quite some time (maybe more than 6 months), and I have to use mirroring with Quicktime as well.  It sometimes starts working again (through no changes of my own) and then it will randomly quit again.  I assume it has something to do with either OS, iOS, or Vysor updates.  Tried all the same things the previous user did.  My coworker is having the same issues as well.   Last time I thought it was an iOS15 issue, but it started working again, only to quit, yet again.  Very frustrating.

**Comment by koush** [comment]
> I have been getting this issue consistently for quite some time (maybe more than 6 months), and I have to use mirroring with Quicktime as well. It sometimes starts working again (through no changes of my own) and then it will randomly quit again. I assume it has something to do with either OS, iOS, or Vysor updates. Tried all the same things the previous user did. My coworker is having the same issues as well. Last time I thought it was an iOS15 issue, but it started working again, only to quit, yet again. Very frustrating.

can you let me know which dongle you are using?

**Comment by jillomonster** [comment]
I have it plugged into USB on my mac.  I've been using your software for a
while with no other bluetooth set up.

**Comment by koush** [comment]
I've added an option to bypass the "connecting" issue in the latest release.

**Comment by pprathyusha** [comment]
Hi
I've got the makerdiary(no case) dongle. https://a.co/d/1O9OvC0. Also is there any particular way i have to attach those two pins on the dongle? 

Copied the file zephyr.uf2 onto the storage to flash the dongle. It rebooted as expected.
Two problems: 
I cannot see the dongle in the iPhones bluetooth listed devices.
Vysor application cannot detect the dongle. It says "Vysor dongle not found" even after clicking refresh multiple times.

FYI: I've tried using MacBook Pro 2019, MacBook M1 and iPhone 14 and iPhone SE.

Please help.
